plasmapy/physics/transport/collisions.py

```diff
@@ -943,24 +969,29 @@ def collision_rate_ion_ion(T_i,
        revised." Naval Research Lab. Report NRL/PU/6790-16-614 (2016).
        https://www.nrl.navy.mil/ppd/content/nrl-plasma-formulary
 
-    .. [3] Callen Chapter 2, http://homepages.cae.wisc.edu/~callen/chap2.pdf
+    .. [3] J.D. Callen, Fundamentals of Plasma Physics draft material,
+       Chapter 2, http://homepages.cae.wisc.edu/~callen/chap2.pdf
 
     Examples
     --------
```

Comment:
It would be good to have one of these examples using a temperature in kelvin. :joy_cat: That way, the users would know that the temperature does not have to be in eV, and it would also provide a test.